### DISCUSSION TITLE
Horizontaler Versatz/Offset Konvoi

### DISCUSSION THREAD

**Discussion by PhilippS1** [Courseplay/Courseplay_FS22#1112] [General]
Hallo zusammen,

bestimmt wurde es schon angesprochen nur finde ich es nicht. 
Funktioniert derzeit der automatischer Versatz noch nicht? Was ich damit meine ist wenn zwei Fahrzeuge im Konvoi fahren kann ich keinen Versatz einstellen da, was ich ja auch muß da die Artbeitsbreit nicht die tatsächliche Breite des Fahzeuges widerspiegelt. Wenn jetzt also meine Schneidwerk zum Beispiel 13,8 Meter breit ist kommt das Fahrzeug in der gegenrichtung nicht an dem folgendem Fahrzeug vorbei. Stelle ich jetzt jedoch beim ersten Fahrzeug (links) eine horizontalen Versatz von 0.2 und beim zweiten Fahrzeug (rechts) einen Versatz von -0.2 ein würden sie zwar aneinander vorbei kommen, jedoch beim Bahn wechsel bleibt der Versatz wie eingestellt.
Eingestellt habe ich Symetrischer Bahnwecheln aktiv.

**Comment by Tensuko** [comment]
Dann hast du die falsche Arbeitsbreite.

**Reply by PhilippS1** [reply]
Die Arbeitsbreite wird ja automatisch erkannt und eingestellt. Daran habe ich nichts geändert.

**Comment by Tensuko** [comment]
Das Problem beim Dreschen ist das Symetrischer Bahnwechsel natürlich die bessere Idee ist damit der Drescht nicht immer wieder mal mit dem Rohr in der Frucht ist...
Trenn dich mal vom klassischen hoch/runter und probiere Beete aus. Dann brauchst du auch keinen symetrischen Bahnen wechsel oder offset, welcher so wie du es dir vorstellst, nicht funktioniert.

**Comment by PhilippS1** [comment]
Habe das Problem auch beim Grubbern. Was meinst du mit Beete wo muss ich das einstellen?

**Reply by Tensuko** [reply]
Im Kursgenerator. Statt hoch/runter.

**Reply by rohne83** [reply]
Das stumpfe hoch/runter ist für Verbände generell herzlich ungeeignet. Beim Dreschen wie Tensuko sagt am besten auf Beete zurückgreifen. Fürs Grubbern und sonstige Bodenbearbeitung mit Konvois ist "kreisförmig" empfehlenswert und auch realitätsnah.

**Comment by PhilippS1** [comment]
Gibt es eigentlich irgendwo ein Wiki das die jeweiligen Einstellungen erklärt? Für Cousreplay 5.0 gab es mal eine sehr schöne Erklärung.

**Reply by Tensuko** [reply]
Ich arbeite zwischendrin an den Einträgen für das Hilfsmenü, vlt kann ich das noch etwas einbauen.
Aber was Generator Settings angeht, hilft ausprobieren und schauen was passiert bei welchen Einstellungen.
